parse_and_import_medicover.py

```
@@ -164,33 +163,67 @@ def main(
+            # flatten nested structure to enable loop below
+            if structure == "nested":
+                variants = []
+                for finding_type in evaluation["variants"]:
+                    for variant in evaluation["variants"][finding_type]["snp"]:
+                        variant["finding_type"] = finding_type
+                        variants.append(variant)
+            else:
+                variants = evaluation["variants"]
+            
+            if len(variants) == 0:
+                print(f"No variants present.")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)